pkg/controllers/managementuser/resourcequota/resource_quota_common.go

```diff
@@ -270,3 +270,18 @@ func convertPodResourceLimitToLimitRangeSpec(podResourceLimit *v32.ContainerReso
 	}
 	return limitRangeSpec, err
 }
+
+func zeroOutResourceQuotaLimit(limit *v32.ResourceQuotaLimit) (*v32.ResourceQuotaLimit, error) {
```

Comment:
This is only used once so it's not really "common", might be more appropriate to put it in resource_quota_sync.go where the original getDefaultQuotaSpec was.

Reply:
Agreed!